### PR TITLE
docs: tweak version select option order

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

修改后
image
同3.x效果
image

### <samp>🤖 Generated by Copilot at e4ff147</samp>

* Add `docNewVersions` property to `HeaderProps` interface and `versions` array to enable switching to new versions of documentation from header component ([link](https://github.com/ant-design/ant-design/pull/42564/files?diff=unified&w=0#diff-82db02ea5a7fa23676967ea5a5d93fee900343e3b2b941c824f464ab9060ea9aL35-R35), [link](https://github.com/ant-design/ant-design/pull/42564/files?diff=unified&w=0#diff-82db02ea5a7fa23676967ea5a5d93fee900343e3b2b941c824f464ab9060ea9aR211), `Layout/Header/index.tsx`)

### CODE REVIEW

**Comment by yoyo837** [comment]
一并禁用掉 `declaration-block-no-redundant-longhand-properties` 吧

**Comment by crazyair** [comment]
> 一并禁用掉 `declaration-block-no-redundant-longhand-properties` 吧

什么意思

**Comment by yoyo837** [comment]
stylelint报错导致的CI failed.

File: .stylelintrc.json [hunk 1]
```diff
@@ -7,7 +7,12 @@
   "customSyntax": "postcss-less",
   "plugins": ["stylelint-declaration-block-no-ignored-properties"],
   "rules": {
-    "function-name-case": ["lower", { "ignoreFunctions": ["/colorPalette/"] }],
+    "function-name-case": [
+      "lower",
+      {
+        "ignoreFunctions": ["/colorPalette/"]
```
Comment (afc163) [inline .stylelintrc.json:13]: 这个可以去掉了。

Comment (yoyo837) [reply .stylelintrc.json]: > 这个可以去掉了。

这是4.x

Comment (crazyair) [reply .stylelintrc.json:13]: colorPalette 这个先这样吧，不动了

Comment (afc163) [reply .stylelintrc.json:13]: 哦，原来是 4.x 分钟，那得留着。

Comment (crazyair) [reply .stylelintrc.json]: image
这行代码怎么会报错，怎么会影响到这里

**Review by afc163 — APPROVED** [review]

**Pull request merged**